modules/t/VCFCollection.t

```diff
@@ -503,4 +503,24 @@ if ($dont_fetch_vf_overlaps)
   ok(scalar @{$cons} eq 4, "get consequences from VCF");
 }
 
+# Test get_all_clinical_significance_states() with VCF files
+$coll = $vca->fetch_by_id('ClnSig');
+ok($coll && $coll->isa('Bio::EnsEMBL::Variation::VCFCollection'), "fetch_by_id");
+
+my $temp_clnsig_filename = $coll->filename_template();
+$temp_clnsig_filename =~ s/###t\-root###/$dir/;
+$coll->filename_template($temp_clnsig_filename);
+ok($coll->filename_template =~ /^$dir/, "update filename_template");
+
+$slice = $sa->fetch_by_region('toplevel', 1, 10, 20);
+my $dont_fetch_vf_overlaps=1;
+my $vfs = $coll->get_all_VariationFeatures_by_Slice($slice, $dont_fetch_vf_overlaps);
+
+ok($vfs->[0]->get_all_clinical_significance_states()->[0] eq 'likely benign', 'get_all_clinical_significance_states - obtain single clinical significance entry');
+ok(scalar (@{$vfs->[1]->get_all_clinical_significance_states()}) eq 1, 'get_all_clinical_significance_states - ignore upsupported clinical significance entry');
```

Comment:
@nakib103 reccomended that the this test be changed. Currently, "affects" is set to be ignored when extracting. But this ClinVar cliical significance term will be supported in a future update. I have changed "_Affects" from the test VCF file to another term.